src/playback-watcher.js

```diff
@@ -350,7 +350,9 @@ export default class PlaybackWatcher {
 
     // verify that at least two segment durations have been
     // appended before checking for a gap.
-    const twoSegmentDurations = (this.media().targetDuration - Ranges.TIME_FUDGE_FACTOR) * 2;
+    const targetDuration = this.media().partTargetDuration || this.media().targetDuration;
```

Comment:
This helps skip gaps faster in llhls, mostly this is helping with our bad segment choice at the start of a playlist. It is likely something we want to do anyway though, as we don't want to wait very long to skip gaps in llhls.